.github/workflows/ci.yaml

```diff
@@ -20,3 +23,27 @@ jobs:
       - uses: open-turo/actions-gha/test@v2
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  # lint-oss tries to run this action against open source repositories not in
+  # open-turo to make sure this is usable outside the org.
+  # Repositories added to this list should have pre-commit hooks that don't
+  # require external tooling to be installed.
+  # Repositories added to this list must have passing pre-commit hooks.
+  lint-oss:
+    name: Lint / Open source
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        repos:
+          - tiangolo/fastapi
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ matrix.repos }}
+      - uses: actions/checkout@v4
+        with:
+          path: this
```

Comment:
Nice way around! Seems like it also checks out the current branch, not sure how it finds it but that seems to work 👌 

Should we update `this` to be `actions-python` to make this clearer?